### ISSUE TITLE
[Beta] Applied Accessibility - Tests fails to check for the closing </a> tag

### ISSUE THREAD

**Issue by Asjas** [freeCodeCamp/freeCodeCamp#16424]
#### Challenge Name

https://beta.freecodecamp.org/en/challenges/applied-accessibility/give-links-meaning-by-using-descriptive-link-text

#### Issue Description

This challenge has a issue where it isn't checking if the closing `</a>` is being used. You can replicate this by moving the opening `<a>` tag and deleting the closing `</a>` tag. The test will pass but it shouldn't do so.

The test needs to check if the closing `</a>` tag is also being used.

#### Browser Information

* Browser Name, Version: Chrome, Version 63.0.3239.108 (Official Build) (64-bit)
* Operating System: Windows 10 Pro
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```html
<body>
  <header>
    <h1>Deep Thoughts with Master Camper Cat</h1>
  </header>
  <article>
    <h2>Defeating your Foe: the Red Dot is Ours!</h2>
    <p>Felines the world over have been waging war on the most persistent of foes. This red
 nemesis combines both cunning stealth and lightening speed. But chin up, fellow 
fighters, our time for victory may soon be near. Click here for <a href="">information
 about batteries</p>
  </article>
</body>
```
#### Screenshot

![achor](https://user-images.githubusercontent.com/3828967/34639888-3becc616-f2f1-11e7-8374-4f5f0c9c6610.jpg)

**Comment by vkWeb** [comment]
Confirmed!

**Comment by ooade** [comment]
That's an issue on [this line](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/seed/challenges/01-responsive-web-design/applied-accessibility.json#L972). A direct comparison should fix that... Probably
```js
$('a').text().trim() == 'information about batteries'
```

**Comment by 0x0936** [comment]
I have a branch with this issue fixed. If no one else is working on it, I'll submit a PR with the fix being what @ooade suggested.

**Comment by vkWeb** [comment]
@0x0936 Go ahead if you have the fix ready.
